src/containers/TopicPage/TopicPage.jsx

```diff
@@ -59,98 +54,80 @@ const getTitle = (article, topic) => {
   return '';
 };
 
+const transformData = data => {
```

Comment:
Used more than once, should this be placed as util or something similar?

Reply:
They are similar now, but will probably diverge in the future.

Reply:
Ah, never mind then.